### DISCUSSION TITLE
missing .dll for release 1.1d and 1.1e?

### DISCUSSION THREAD

**Discussion by ebeetham** [CyprienBosserelle/xbeach_gpu#58] [General]
Is the pre-compiled release for 1.1d (17 June) and 1.1e (12 July) missing Codart64_101.dll from the upload?

**Comment by CyprienBosserelle** [comment]
I made a new release with the dlls. See 
https://github.com/CyprienBosserelle/xbeach_gpu/releases/tag/v1.1f-2
